File: apps/polyglot-sample/sample.py
def answer() -> int:
    return 42

def main() -> None:
    pass


class Counter:
    value: int

    def __init__(self, start: int) -> None:
        self.value = start

    def inc(self) -> int:
        value = self.value + 1
        self.value = value
        return value


double = lambda n: n * 2

File: apps/polyglot-sample/sample.js
function answer() {
  return 42;
}

function main() {}

class Counter {
  constructor(start) {
    this.value = start;
  }
  inc() {
    this.value = this.value + 1;
    return this.value;
  }
}

const double = function(n) {
  return n * 2;
};

const triple = (n) => {
  return n * 3;
};
